rules_bot.py

```diff
@@ -130,35 +213,35 @@ def off_on_topic(bot, update, groups):
             else:
                 name = reply.from_user.first_name
 
-            replied_message_text = reply.text
+            replied_message_text = convertformatting(reply.text, reply.entities)
             replied_message_id = reply.message_id
 
-            text = (f'{name} [wrote](t.me/pythontelegrambotgroup/{replied_message_id}):\n'
+           text = (f'{name} <a href = "t.me/pythontelegrambotgroup/{replied_message_id}">wrote</a>:\n'
```

Comment:
You removed one of spaces. Python will throw indentation error

Had you tried to run your code?